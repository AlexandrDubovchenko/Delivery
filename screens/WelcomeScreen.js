import React from 'react';
import { StyleSheet, View } from 'react-native';
import { SignIn } from '../components/welcomeComponents/SignIn';
import { SignUp } from '../components/welcomeComponents/SignUp';
import { WelcomeHeader } from '../components/welcomeComponents/WelcomeHeader';
export const WelcomeScreen = () => {
  return (
    <View style={styles.container}>
      <WelcomeHeader/>
      <View style={styles.content}>
        <SignIn/>
        <SignUp/>
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
  },
  content: {
    flex: .5,
    alignItems: "center",
    justifyContent: "center",
    backgroundColor: "#006400"
  },
});