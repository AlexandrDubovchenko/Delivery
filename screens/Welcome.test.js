import React from 'react';
import WelcomeScreen from './Welcome';
import TestRenderer from 'react-test-renderer';

jest.mock('react-native-gesture-handler', () => {});

test('screen should render without error', () => {
  
  const signUpTest = TestRenderer.create(<WelcomeScreen />);
  expect(signUpTest.toJSON()).toMatchSnapshot();
  
});