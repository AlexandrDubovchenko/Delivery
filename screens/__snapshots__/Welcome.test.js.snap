// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screen should render without error 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFD700",
        "flex": 1,
        "justifyContent": "space-around",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#006400",
          "fontSize": 50,
        }
      }
    >
      Hunter Delivery
    </Text>
    <Text
      style={
        Object {
          "color": "#006400",
          "fontSize": 20,
        }
      }
    >
      Welcome to the Hunter Delivery
    </Text>
  </View>
  <View
    accessibilityLabel="test"
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#006400",
        "flex": 0.5,
        "justifyContent": "center",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#FFD700",
          "borderRadius": 50,
          "marginBottom": 20,
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Text>
        Sign In
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#FFD700",
          "borderRadius": 50,
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Text>
        Sign Up
      </Text>
    </View>
  </View>
</View>
`;
